native/logik_simulation/src/ffi.rs

```diff
@@ -36,21 +36,21 @@ pub extern "C" fn exit(data: *mut Data) {
 }
 
 #[no_mangle]
-pub extern "C" fn add_subnet(data: *mut Data, id: usize) -> bool {
+pub extern "C" fn add_subnet(data: *mut Data, id: i32) -> bool {
     let data = unsafe { &mut *data};
     
-    data.add_subnet(id)
+    data.add_subnet(id as usize)
```

Comment:
This cast could easily be avoided if the datastructure stores the subnet ID:s as `i32` instead of as `usize`.